client/src/partials/patients/registry/modal.js

```diff
@@ -0,0 +1,100 @@
+angular.module('bhima.controllers')
+.controller('PatientRegistryModalController', PatientRegistryModalController);
+
+PatientRegistryModalController.$inject = [
+  '$uibModalInstance', 'InventoryService', 'PatientService', 'util'
+];
+
+function PatientRegistryModalController( $uibModalInstance, Inventory, patients, util) {
+  var vm = this;
+
+  vm.period = {
+    today : {
+      cacheKey : 'today',
+      translateKey : 'FORM.BUTTONS.TODAY',
+      id : 'today'
+    },
+    week : {
+      cacheKey : 'week',
+      translateKey : 'FORM.BUTTONS.THIS_WEEK',
+      id : 'week'
+    },
+    month : {
+      cacheKey : 'month',
+      translateKey : 'FORM.BUTTONS.THIS_MONTH',
+      id : 'month'
+    },
+    year : {
+      cacheKey : 'year',
+      translateKey : 'FORM.BUTTONS.THIS_YEAR',
+      id : 'year'
```

Comment:
Thank you for clarifying.  From [this line](https://github.com/IMA-WorldHealth/bhima-2.X/pull/403/files#diff-7affaeca06dfed7a1ae3d4f1d6df6c00R22) I see that the HTML element id is set to the `cacheKey` property, instead of the `id` property.  Is it used somewhere else that I cannot find?